tests/test_github.py

```diff
@@ -0,0 +1,746 @@
+        httpretty.register_uri(httpretty.GET,
+                               GITHUB_ISSUES_URL,
+                               body=issue_1,
+                               status=200,
+                               forcing_headers={
+                                    'X-RateLimit-Remaining': '4986',
+                                    'Link': '<'+GITHUB_ISSUES_URL+'/?&page=2>; rel="next", <'+GITHUB_ISSUES_URL+'/?&page=3>; rel="last"'
+                               })
+        httpretty.register_uri(httpretty.GET,
+                               GITHUB_ISSUES_URL+'/?&page=2',
+                               body=issue_2,
+                               status=200,
+                               forcing_headers={
+                                    'X-RateLimit-Remaining': '4985'
+                               })
+
+        client = GitHubClient("zhquan_example", "repo", "aaa", None)
+        issues = [issues for issues in client.get_issues()]
+
+        self.assertEqual(len(issues), 2)
+        self.assertEqual(issues[0], issue_1)
+        self.assertEqual(issues[1], issue_2)
+
+        # Check requests
+        expected = {
+                     'per_page': ['30'],
+                     'page': ['2'],
+                     'state': ['all'],
+                     'direction': ['asc'],
+                     'sort': ['updated']
+                   }
+                   
+        self.assertDictEqual(httpretty.last_request().querystring, expected)
+        self.assertEqual(httpretty.last_request().headers["Authorization"], "token aaa")
+
+    @httpretty.activate
+    def test_get_empty_issues(self):
+        """ Test when issue is empty API call """
+
+        issue = read_file('data/github_empty_request')
+
+        httpretty.register_uri(httpretty.GET,
+                               GITHUB_ISSUES_URL,
+                               body=issue, status=200,
+                               forcing_headers={
+                                    'X-RateLimit-Remaining': '4986'
+                               })
+
+        client = GitHubClient("zhquan_example", "repo", "aaa", None)
+
+        raw_issues = [issues for issues in client.get_issues()]
+        self.assertEqual(raw_issues[0], '[]\n')
+
+        # Check requests
+        expected = {
+                     'per_page': ['30'],
+                     'state': ['all'],
+                     'direction': ['asc'],
+                     'sort': ['updated']
+                   }
+
+        self.assertDictEqual(httpretty.last_request().querystring, expected)
+        self.assertEqual(httpretty.last_request().headers["Authorization"], "token aaa")
+
+    @httpretty.activate
+    def test_get_user(self):
+        """ Test get_user API call """
+
+        login = read_file('data/github_login')
+
+        httpretty.register_uri(httpretty.GET,
+                               GITHUB_LOGIN_URL,
+                               body=login, status=200)
+        httpretty.register_uri(httpretty.GET,
```

Comment:
You can remove this because you're only testing the result of calling to `get_user()` method. You can also remove the calls to `get_issues()` from below.